src/library/coap.cpp

```diff
@@ -521,7 +522,8 @@ void Coap::HandleRequest(const Request &aRequest)
 exit:
     if (error != Error::kNone)
     {
-        LOG_INFO("CoAP: handle request failed: %s", ErrorToString(error));
+        LOG_INFO(LOG_REGION_COAP, "server(={}) handle request failed: %s", static_cast<void *>(this),
```

Comment:
why is there a mixed style of `{}` and `%s`

Reply:
`%s` is the stale format, will fix.